3.11/graphs-pregel-algorithms.md

```diff
@@ -127,26 +127,45 @@ pregel.start("scc", "graphname", { resultField: "component_strong" });
 ### Hyperlink-Induced Topic Search (HITS)
 
 HITS is a link analysis algorithm that rates Web pages, developed by
-Jon Kleinberg. The algorithm is also known as _Hubs and Authorities_.
+Jon Kleinberg in J. Kleinberg,
+[Authoritative sources in a hyperlinked environment](http://www.cs.cornell.edu/home/kleinber/auth.pdf),
+Journal of the ACM. 46 (5): 604–632, 1999. The algorithm is also known as _Hubs and Authorities_.
 
-The idea behind Hubs and Authorities comes from the typical structure of the web:
-Certain websites known as hubs, serve as large directories that are not actually
-authoritative on the information that they hold. These hubs are used as
-compilations of a broad catalog of information that leads users direct to other
-authoritative webpages.
+The idea behind hubs and authorities comes from the typical structure of the early web:
+Certain websites, known as hubs, serve as large directories that are not actually
```

Comment:
```suggestion
certain websites, known as hubs, serve as large directories that are not actually
```

Reply:
I may be wrong, but as far as I know, after a colon, we start from the capital letter if what is before the colon refers to more than one sentence. In this case, we have two.

Reply:
The style guide we use recommends using a lowercase for the first word that follows a colon. We can also drop the colon and have separate sentences.
https://developers.google.com/style/colons#colons-within-sentences 
https://developers.google.com/style/capitalization#capitalization-and-colons